chia/cmds/cmds_util.py

```diff
@@ -103,9 +103,9 @@ async def get_any_service_client(
         if connected:
             yield node_client, config
         else:
-            yield None, config
+            raise CliRpcConnectionError
     except Exception as e:  # this is only here to make the errors more user-friendly.
-        if not consume_errors:
+        if not consume_errors or isinstance(e, CliRpcConnectionError):
```

Comment:
this is a bit hacky, but I can't think of an obvious improvement. I imagine `validate_client_connection()` itself can throw other exceptions too, that we want to print out. Otherwise the scope of the `try` block could just be made smaller

Reply:
yeah we only really want to catch a timeout error but throw anything else